src/renderer/Core/Settings/Pages/General/General.tsx

```diff
@@ -13,6 +14,7 @@ export const General = () => {
                 <Language />
                 <HotKey />
                 <Autostart />
+                <DockSettings />
```

Comment:
I liked the previous approach better. Please render this component only on macOS.

Reply:
This requires the `useContextBridge` hook to be on the `General` component, which I wanted to avoid to keep the component clean. This is why I added a wrapper component that just renders nothing on systems other than macOS. I can change it, but thought it would make things a bit more organized

Reply:
I don't think that having the contextBridge in `General.tsx` hurts. A component should generally not care about when it should be rendered or not. It's the parent's component responsibility to decide when it should render specific settings or not. There are already similar examples, take a look at `Window.tsx`.

Reply:
Agree. Changed it